### ISSUE TITLE
Should we implement pbkdf2 for LibraWallet account generator?

### ISSUE THREAD

**Issue by totiz** [bandprotocol/pylibra#1]
As I tested import mnemonic from Libra cli it got a different address (but all the signing func work properly).

I think the problem is how the account is generated. 
**Refer to this** for seed generator  https://github.com/libra/libra/blob/master/client/libra_wallet/src/key_factory.rs#L172

So I think if we implement seed / pbkdf2 in the same way as Libra cli does, we can guarantee that user can use the same mnemonic to access to the same account.

**Comment by sorawit** [comment]
Yeah that is still an open issue. As specified in README, "Note that PyLibra's mnemonic scheme is not similar to that of Libra's CLI, so you cannot import mnemonic between the two libraries (yet)."

I will look into that probably this weekend. Or you can open a pull request if you feeling writing and hacking in Python (:

**Comment by totiz** [comment]
@sorawit yeah that sounds great, thanks.
